server/stt/stt_planning_ml.py

```diff
@@ -124,7 +139,227 @@ async def get_coverage_details(
+        try:
+            vocab_service = get_resource_service("vocabularies")
+            coverage_status_vocab = vocab_service.find_one(
+                req=None, _id="newscoveragestatus"
+            )
+
+            if coverage_status_vocab and "items" in coverage_status_vocab:
+                for item in coverage_status_vocab["items"]:
+                    if item.get("qcode") == qcode and item.get("is_active", True):
+                        return {
+                            "qcode": item["qcode"],
+                            "name": item.get("name", ""),
+                            "label": item.get("label", ""),  # Use label from vocabulary
+                        }
+        except Exception as e:
+            logger.warning(
+                f"Failed to get coverage status from vocabulary for {qcode}: {e}"
+            )
+
+        return None
+
+    def parse_picture_type(self, subject_elt: Element, coverage: Dict[str, Any]):
+        """Parse picture type from sttimagetypename subject"""
+        qcode = subject_elt.get("qcode", "")
+        # Convert sttimagetypename:XX to sttimage:XX
+        picture_qcode = qcode.replace("sttimagetypename:", "sttimage:")
+
+        # Get picture type from vocabulary and store in planning.subject
+        picture_type = self.get_picture_type_from_vocabulary(picture_qcode)
+        if picture_type:
+            coverage["planning"]["subject"].append(picture_type)
+
+    def get_picture_type_from_vocabulary(self, qcode: str) -> Optional[Dict[str, Any]]:
+        """Get picture type from sttimagetype vocabulary"""
+        try:
+            vocab_service = get_resource_service("vocabularies")
+            picture_type_vocab = vocab_service.find_one(req=None, _id="sttimagetype")
+
+            if picture_type_vocab and "items" in picture_type_vocab:
+                for item in picture_type_vocab["items"]:
+                    if item.get("qcode") == qcode and item.get("is_active", True):
+                        return {
+                            "qcode": item["qcode"],
+                            "name": item.get("name", ""),
+                            "scheme": "sttimagetype",
+                        }
+        except Exception as e:
+            logger.warning(
+                f"Failed to get picture type from vocabulary for {qcode}: {e}"
+            )
+
+        return None
+
+    def parse_photographer_awareness(
+        self, subject_elt: Element, coverage: Dict[str, Any]
+    ):
+        """Parse photographer awareness from sttphotoaware subject"""
+        qcode = subject_elt.get("qcode", "")
+
+        # Only store if photographer knows (sttphotoaware:2) or doesn't know (sttphotoaware:1)
+        # Don't store for sttphotoaware:0 (not set)
+        if qcode in ["sttphotoaware:1", "sttphotoaware:2"]:
+            photographer_awareness = self.get_photographer_awareness_from_vocabulary(
+                qcode
```

Comment:
[qcode](https://github.com/superdesk/superdesk-stt/blob/develop/server/data/vocabularies.json#L875) is either `yes` or `no`